### PR TITLE
Update Readme Build Instructions for Debian

### PR DESCRIPTION
**Pull request by Xyaren**
Update Readme Build part to include Debian dependencies.
Based on #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +37,20 @@ I do **not** support or endorse these repositories, **use at your own risk.**
...
 
-```console
-$ make build
-$ sudo make install
-```
+2. **Debian only:** Make sure `libgtk-3-dev` is installed, if not install it with `sudo apt install libgtk-3-dev`.
 
-This install to `/usr/local/`.
+3. Build ```$ make build```
+4. Install `$ sudo make install`
```
Comment (lunaryorn) [inline README.md:45]: Please remove the leading `$` here; it no longer resembles console input so the `$` which indicates the prompt no longer makes sense.

File: README.md [hunk 2]
```diff
@@ -37,19 +37,20 @@ I do **not** support or endorse these repositories, **use at your own risk.**
...
+   
+   This installs to `/usr/local/`.
 
-**Note:** You really do need to install as `root`, system-wide.
-A per-user installation to `$HOME` does not work as of Gnome 40, because Gnome shell doesn't load search providers from `$HOME` (see <https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3060>).
+   **Note:** You really do need to install as `root`, system-wide.
+   A per-user installation to `$HOME` does not work as of Gnome 40, because Gnome shell doesn't load search providers from `$HOME` (see <https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3060>).
 
-To uninstall use `sudo make uninstall`.
+## Uninstallation
```
Comment (lunaryorn) [inline README.md:52]: Please add a blank line after the headline, for consistency with the other headlines.

File: README.md [hunk 3]
```diff
@@ -37,19 +37,20 @@ I do **not** support or endorse these repositories, **use at your own risk.**
...
 ### From source
 
-Install [rust](https://www.rust-lang.org/tools/install) then run
+1. Install [rust](https://www.rust-lang.org/tools/install)
 
-```console
-$ make build
-$ sudo make install
-```
+2. **Debian only:** Make sure `libgtk-3-dev` is installed, if not install it with `sudo apt install libgtk-3-dev`.
```
Comment (lunaryorn) [inline README.md:42]: I'm not a native speaker, but shouldn't this be a semicolon?

**Review by lunaryorn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Xyaren) [reply README.md]: Removed them

File: README.md (hunk 2 above)
Comment (Xyaren) [reply README.md:52]: Sure

File: README.md (hunk 3 above)
Comment (Xyaren) [reply README.md]: I'm not a native speaker either :D
fixed :)

**Comment by lunaryorn** [comment]
Thanks 🙏

**Pull request merged**